[storage] Adding a section about recovering osts files (#349)
OfficeDev/office-scripts-docs: docs/overview/script-storage.md

Scripts that are shared with one of your workbooks remain in the script creator's OneDrive. They are not copied to any of your local or OneDrive folders when you run the shared script in Excel. The **Make a Copy** button of the Code Editor saves a separate copy of the script in your OneDrive. Changes to the copy don't affect the original script.

### Restore deleted scripts

When you delete a script in Excel, it goes to your OneDrive recycle bin. To restore a deleted script, follow the steps listed in [Restore deleted files or folders in OneDrive](https://support.microsoft.com/office/restore-deleted-files-or-folders-in-onedrive-949ada80-0026-4db3-a953-c99083e6a84f). Restoring an **.osts** file returns it to the **All scripts** list.

> A deleted script in unshared with the workbook. When you restore a script, it does **not** retain it's script access. You will need to share the script again.

Reviewer: Fixing two typos. 
```suggestion
A deleted script is unshared with the workbook. When you restore a script, it does **not** retain its script access. You will need to share the script again.
```
